### PR TITLE
Add place_id to index table schema.

### PR DESCRIPTION
**Pull request by geening**

### CODE REVIEW

**Comment by owahltinez** [comment]
Can you please add the field to the documentation too? See this for the text that we should use: https://github.com/GoogleCloudPlatform/covid-19-open-data/commit/b195658d5dd4d880d239c530210e9e918d2d1f9e#diff-469c801c330ca0ed15a7f54a6bbe07a66050d01d3fb7466653a8a6896748732cR76

**Comment by geening** [comment]
> Can you please add the field to the documentation too? See this for the text that we should use: [b195658#diff-469c801c330ca0ed15a7f54a6bbe07a66050d01d3fb7466653a8a6896748732cR76](https://github.com/GoogleCloudPlatform/covid-19-open-data/commit/b195658d5dd4d880d239c530210e9e918d2d1f9e#diff-469c801c330ca0ed15a7f54a6bbe07a66050d01d3fb7466653a8a6896748732cR76)

Done.

File: docs/table-index.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ This table can be found at the following URLs depending on the choice of format:
 | Name | Type | Description | Example |
 | ---- | ---- | ----------- | ------- |
 | **key** | `string` | Unique string identifying the region | US_CA_06001 |
+| **place_ID** | `string` | A textual identifier that uniquely identifies a place in the Google Places database and on Google Maps ([details](https://developers.google.com/places/web-service/place-id))| ChIJd_Y0eVIvkIARuQyDN0F1LBA |
```
Comment (owahltinez) [inline docs/table-index.md:16]: Can you please fix the capitalization? I think the name should be `place_id`

**Review by owahltinez — CHANGES_REQUESTED** [review]

File: docs/table-index.md (hunk 1 above)
Comment (owahltinez) [reply docs/table-index.md:16]:
```suggestion
| **place_id** | `string` | A textual identifier that uniquely identifies a place in the Google Places database and on Google Maps ([details](https://developers.google.com/places/web-service/place-id))| ChIJd_Y0eVIvkIARuQyDN0F1LBA |
```

Comment (geening) [reply docs/table-index.md]: Done.

**Review by owahltinez — APPROVED** [review]
Thank you!

**Pull request merged**